rpc/handlers.go

```diff
@@ -1340,6 +1340,15 @@ func (h *Handler) EstimateMessageFee(msg MsgFromL1, id BlockID) (*FeeEstimate, *
 	return &estimates[0], nil
 }
 
+func (h *Handler) LegacyEstimateMessageFee(msg MsgFromL1, id BlockID) (*FeeEstimate, *jsonrpc.Error) { //nolint:gocritic
+	estimate, rpcErr := h.EstimateMessageFee(msg, id)
```

Comment:
If you call LegacyEstimateMessageFee here, you wont need to explicitly nillify Unit.